File: Pset6/mario/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h > 0 and h <= 8:
        break

for i in range(1, h + 1):
    hashes = i
    spaces = h - hashes
    print(" " * spaces, end="")
    print("#" * hashes, end="")
    print(" " * 2, end="")
    print("#" * hashes, end="")
    print()

File: Pset1/Mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
        if (n > 8){
            n = -1;
        }    
    }
    while (n < 1);
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j <= n - (i + 2) ; j++)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        for (int s = 0; s < 2; s++)
        {
            printf(" ");
        }
        for (int l = 0; l <= i; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

